picturesafe-search-spring-boot-autoconfigure/src/main/java/de/picturesafe/search/autoconfigure/PicturesafeSearchAutoQueryConfiguration.java

```diff
@@ -58,7 +58,7 @@ public QueryConfiguration queryConfiguration() {
     }
 
     @Bean
-    @ConditionalOnMissingBean
+    @ConditionalOnMissingBean(name = "queryFactories")
     public List<QueryFactory> queryFactories(FulltextQueryFactory fulltextQueryFactory,
                                              OperationExpressionQueryFactory operationExpressionQueryFactory,
                                              NestedQueryFactory nestedQueryFactory) {
```

Comment:
Please extend this method to the current implementation in DefaultQueryConfiguration. The order of the query factories is important for the list. We should think about refactoring this.

Reply:
Added RelevanceSortQueryFactory

Reply:
What about FindAllQueryFactory? With "current implementation" i meat the pull request i assigned to you yesterday.

Reply:
Merged open pull request from picturesafe-search and added FindAllQueryFactory.